### ISSUE TITLE
Add Bybit V5 API

### ISSUE THREAD

**Issue by Cypher01** [contek-io/io.contek.invoker#73]
Bybit has added a new API version V5, that unifies all old APIs (Spot, Derivatives, Options).

https://bybit-exchange.github.io/docs/v5/upgrade-guide

It's already helpful that the old derivatives API module "bybit-api" was renamed to "bybitinverse-api".

A new API module "bybit-api" could be added, implementing the Bybit V5 API. This preserves backwards compatibility for everyone using the existing modules.

**Comment by Krzysztof-Konopa** [comment]
Is this support for v5 api has been already provided ?
Recently I've got an email from Bybit that they are decomissioning V1 endpoints since October this year.
Here is a fragment:

Please be aware of the following key dates for the discontinuation of legacy services:

1. Public API Endpoints and Websocket Streams: Offline from Oct 16, 2023
Affected services include but are not limited to USDC Option V1, USDC Perpetual V1, and Spot V1 market data endpoints, among others.

2. Private API Endpoints and Websocket Streams: Offline from Oct 30, 2023
Impacted services include account data endpoints for USDC Option V1, USDC Perpetual V1, and Spot V1, among other services.

For a detailed list of affected and impacted services, please refer to our [official announcement]

**Comment by Cypher01** [comment]
Good point. All Bybit API endpoints implemented here are V1 or V2, both are going to be discontinued soon.

Full announcement:
https://announcements.bybit.com/en-US/article/bybit-openapi-services-transition-from-legacy-version-to-new-v5-api-blt25b43a5738c00765/

This means V5 needs to be implemented asap, otherwise this library will not work for Bybit anymore.
